pkg/drivers/kic/kic.go

```diff
@@ -81,6 +83,17 @@ func (d *Driver) Create() error {
 		APIServerPort: d.NodeConfig.APIServerPort,
 	}
 
+	if gateway, err := oci.CreateNetwork(d.OCIBinary, d.NodeConfig.ClusterName); err != nil {
+		out.WarningT("Unable to create dedicated network, This might result in cluster IP change after restart. {{.error}}", out.V{"error": err})
```

Comment:
nit: `This` should be lowercase -- `this`
I would also do a colon after restart, so it looks like `restart: {{.error}}`